pkg/virt-operator/creation/components/crds.go

```diff
@@ -513,6 +513,22 @@ func NewPrometheusRuleSpec(ns string) *promv1.PrometheusRuleSpec {
 							"summary": "More than one virt-api should be running if more than one worker nodes exist.",
 						},
 					},
+					{
+						Record: "num_of_kvm_available_nodes",
+						Expr:   intstr.FromString("num_of_allocatable_nodes - count(kube_node_status_allocatable{resource=\"devices_kubevirt_io_kvm\"} == 0)"),
+					},
+					{
+						Alert: "LowKVMNodesCount",
+						Expr:  intstr.FromString("(num_of_allocatable_nodes > 1) and (num_of_kvm_available_nodes < 3)"),
```

Comment:
Shouldn't it be `(num_of_kvm_available_nodes < 2)` ?

Reply:
Yes. Changed.